### PR TITLE
[SELS-TASK] BE-Activity-Logs-API

### PR DESCRIPTION
**Pull request by ragg09-sunasterisk**
### Asana Link
https://app.asana.com/0/1203341829928748/1203341829928703/f

### Commands
```
cd backend
php artisan serve
cd frontend
npm start
```
### Pre-conditions
none

### Expected Output
- [X] Be able to return Activity Logs

### Notes
-it returns static data for the mean time
- I will update it along with the task [BE] Get Category and Words API request

### Screenshots
image

### CODE REVIEW

File: be/app/Http/Controllers/ActivityLogsController.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $activities = array(
+            [
```
Comment (escbooster12-dev) [inline be/app/Http/Controllers/ActivityLogsController.php:17]: - why this one naka static?

File: be/database/migrations/2022_11_11_021215_create_activity_logs_table.php [hunk 2]
```diff
@@ -15,7 +15,7 @@ public function up()
     {
         Schema::create('activity_logs', function (Blueprint $table) {
             $table->id();
-            $table->string('loggable_id');
+            $table->string('loggale_id');
```
Comment (escbooster12-dev) [inline be/database/migrations/2022_11_11_021215_create_activity_logs_table.php]: - do not change existing migrations (from the dev branch). Instead generate a new migration.
- `loggable_id` should be `unsignedBigInteger`

File: be/routes/api.php [hunk 3]
```diff
@@ -32,4 +33,5 @@
     Route::resource('lessons', LessonController::class);
     Route::resource('words', WordController::class);
     Route::resource('users', UserController::class);
+    Route::resource('logs', ActivityLogsController::class);
```
Comment (escbooster12-dev) [inline be/routes/api.php]: - activity_logs

File: be/app/Http/Controllers/ActivityLogsController.php (hunk 1 above)
Comment (ragg09-sunasterisk) [reply be/app/Http/Controllers/ActivityLogsController.php:17]: I asked you about it sir on the meeting, since I still dont have a data coming from user_words activity logs, and you said make a static data for a meantime then go back if okay na ang data

Comment (escbooster12-dev) [reply be/app/Http/Controllers/ActivityLogsController.php:17]: do we have separate task for this one on when could this be updated? reply here the asana link :man:

File: be/database/migrations/2022_11_11_021215_create_activity_logs_table.php [hunk 4]
```diff
@@ -15,7 +15,7 @@ public function up()
     {
         Schema::create('activity_logs', function (Blueprint $table) {
```
Comment (escbooster12-dev) [inline be/database/migrations/2022_11_11_021215_create_activity_logs_table.php]: - undo this changes.

File: be/database/migrations/2022_11_22_103509_update_loggable_id_to_unsignedbiginteger_type_in_activity_logs_table.php [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('activity_logs', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger("loggable_id");
+            $table->string('loggable_type');
+            $table->string('title');
+            $table->timestamps();
```
Comment (escbooster12-dev) [inline be/database/migrations/2022_11_22_103509_update_loggable_id_to_unsignedbiginteger_type_in_activity_logs_table.php]: - incorrect blueprint. Please research how to update migration column types.

**Review by escbooster12-dev — APPROVED** [review]
Iteration: 5
Comments: 5
